bench/shelve-bench.py

```diff
@@ -64,8 +64,8 @@ def createFile(filename, totalrows, recsize):
     # Get the record object associated with the new table
     if recsize == "big":
         d = Big()
-        arr = NA.array(NA.arange(32), type=NA.Float64)
-        arr2 = NA.array(NA.arange(32), type=NA.Float64)
+        arr = np.array(np.arange(32), type=np.float64)
```

Comment:
First I thought, that this could be written `np.arange(32, type=np.float64)` directly, but `type` is actually an invalid argument in both cases. It should read `dtype`.

@avalentino what's the state of the `bench` scripts? They are not covered by tests.

Reply:
Personally I have not put too much attention in keeping them up to date.
Probably we could add a script that executes all of them in CI.